README.rst

```diff
@@ -1,69 +1,105 @@
-===========  =================================================================================================================================
-Info         Contains a Python API for interacting with NI-DAQmx. See `GitHub <https://github.com/ni/nidaqmx-python/>`_ for the latest source.
-Author       National Instruments
-===========  =================================================================================================================================
++------------+-----------------------------------------------------------+
+| **Info**   | Contains a Python API for interacting with NI-DAQmx. See  | 
+|            | `GitHub <https://github.com/ni/nidaqmx-python/>`_ for the |
+|            | latest source.                                            |
++------------+-----------------------------------------------------------+
+| **Author** | National Instruments                                      |
++------------+-----------------------------------------------------------+
+
+.. contents:: Table of Contents
+   :depth: 1
+   :backlinks: none
 
 About
 =====
 
 The **nidaqmx** package contains an API (Application Programming Interface)
-for interacting with the NI-DAQmx driver. The package is implemented in Python.
-The package is implemented as a complex,
-highly object-oriented wrapper around the NI-DAQmx C API using the
+for interacting with the NI-DAQmx driver.
+
+Documentation
+-------------
+
+You can find the latest API documentation for the **nidaqmx** package on
+`Read the Docs <http://nidaqmx-python.readthedocs.io/en/latest>`_.
+
+Refer to the `NI-DAQmx Help <http://digital.ni.com/express.nsf/bycode/exagg4>`_
+for API-agnostic information about NI-DAQmx or measurement concepts. NI-DAQmx
+Help also installs locally with the full version of NI-DAQmx.
+
+Implementation
+--------------
+
+The package is implemented in Python as a complex, highly object-oriented
+wrapper around the NI-DAQmx C API using the
 `ctypes <https://docs.python.org/2/library/ctypes.html>`_ Python library.
 
+Supported NI-DAQmx Driver Versions
+----------------------------------
+
 **nidaqmx** supports all versions of the NI-DAQmx driver that ships with the C
```

Comment:
Subject/verb agreement: "versions... that ships with...".

Consider replacing "ships with" with "include".

What versions of the NI-DAQmx driver do not ship with the C API? It has been there since the first DAQmx release. Is this about runtime/config distributions?

Reply:
Runtime ships with it, too... so yeah, I'll just remove this.